zt_frontend/src/App.vue

```diff
@@ -115,6 +123,7 @@
         :notebook="notebook"
         :completions="completions"
         @runCode="runCode"
+        @runAllCode="runAllCode"
```

Comment:
Was it necessary to add another method? Was just making a call to runCode with empty originId not working? You would have to adjust the runCode method to accept this but not sure if another method is necessary

Reply:
Also this is for emitted events so should not be needed here. Only on the button

Reply:
Removed runAllCode